src/index.jsx

```diff
@@ -118,95 +135,61 @@ export default class UIIdentificationCard extends React.Component {
   }
 
   /**
-   * @getter {String} path
-   */
-  get path() {
-    return [
-      'M165.845472,0',
-      'L227.99565,0',
-      'C232.416331,0',
-      '236,3.57268443',
-      '236,7.99770351',
-      'L236,141.002296',
-      'C236,145.419306',
-      '232.410871,149',
-      '227.99565,149',
-      'L74.7409935,149',
-      'C81.7669784,130.92177',
-      '98.4541553,89.6860279',
-      '117.435739,54.7599123',
-      'C134.773927,22.8576428',
-      '154.983589,6.85502733',
-      '165.845472,-4.88498131e-15 Z',
-    ].join(' ');
-  }
-
-  /**
    * @constructor
    */
   constructor(props) {
     super(props);
     this.state = {
       fillColor: props.fillColor,
       fillOpacity: props.fillOpacity,
-      contentPosition: 0,
+      shadow: 'none',
+      backgroundPosition: '0px 0px'
     };
   }
 
   onMouseOver() {
     this.setState({
       color: '#777777',
       fillOpacity: UIIdentificationCard.defaultProps.fillOpacity - 0.2,
-      contentPosition: 10,
+      shadow: 'drop-shadow(2px 2px 4px rgba(0,0,0,.5))',
+      backgroundPosition: '0px -30px'
     });
   }
 
   onMouseLeave() {
     this.setState({
       fillOpacity: UIIdentificationCard.defaultProps.fillOpacity,
-      contentPosition: 0,
+      shadow: 'none',
+      backgroundPosition: '0px 0px'
     });
   }
 
   background() {
+    const rectStyl = Object.assign({
+      width: this.width + 'px',
+      height: this.height + 'px',
+      background: this.props.color,
+      WebkitFilter: this.state.shadow
+    }, UIIdentificationCard.styles.rect);
```

Comment:
With this call to `Object.assign`, the `UIIdentificationCard.styles.rect` would cover the sets on your first defined object, is that expected?